hooks/post/30-upgrade.rb

```diff
@@ -31,13 +31,15 @@ def import_subscriptions
 end
 
 def elasticsearch_message
-  return true unless Kafo::Helpers.execute('rpm -q elasticsearch')
-
+  if `rpm -q elasticsearch`.gsub!(/\A"|"\Z/, '') != nil
```

Comment:
@stbenjam the return true unless Kafo::Helpers.execute('rpm -q elasticsearch') was always giving back package elasticsearch is not installed, so it threw up the red error. I was trying to suppress that, I tried adding it to Kafo::Helpers.execute but it was not happy with that, that is why I put it in a if statement.